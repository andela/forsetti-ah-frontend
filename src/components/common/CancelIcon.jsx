import React from 'react';
import cancel from '../../assets/svg/cancel.svg';

const CancelIcon = ({ clicked }) => (
  <img className='cancel-icon' src={cancel} alt='Cancel icon' onClick={clicked} />
);

export default CancelIcon;
