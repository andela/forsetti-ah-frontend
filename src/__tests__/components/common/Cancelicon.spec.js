import React from 'react';
import { shallow } from 'enzyme';
import CancelIcon from '../../../components/common/CancelIcon';

describe('Cancel Icon', () => {
    const clicked = jest.fn();
    const event = {
        clicked
    }
  test('should render cancel icon', () => {
    const icon = shallow(<CancelIcon {...event} />);
  });
});

