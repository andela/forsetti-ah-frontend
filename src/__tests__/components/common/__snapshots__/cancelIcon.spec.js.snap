// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CancelIcon  /> renders the CancelIcon  correctly 1`] = `
<img
  alt="Cancel icon"
  className="cancel-icon"
  src={
    Object {
      "canInstrument": true,
      "createTransformer": [Function],
      "getCacheKey": [Function],
      "process": [Function],
    }
  }
/>
`;
